service_test.go

```diff
@@ -133,3 +138,30 @@ func TestHandleCommandUnknownInput(t *testing.T) {
 	checkStandardOutputStatus(t, err)
 	checkUnknownCommand(t, captured)
 }
+
+// TestInitInfoLog check the function initInfoLog
+func TestInitInfoLog(t *testing.T) {
+	buf := new(bytes.Buffer)
+	log.Logger = zerolog.New(buf)
+
+	expectedString := "*** message ***"
+	main.InitInfoLog(expectedString)
+
+	logContent := buf.String()
+	if !strings.Contains(logContent, expectedString) {
```

Comment:
I think assert.Contain would also work here https://godoc.org/github.com/stretchr/testify/assert#Contains but it's a nitpick :)